Basic/ClientDriven/Assets/Scripts/ServerScoreReplicator.cs

```diff
@@ -1,9 +1,10 @@
 using Unity.Netcode;
-using Unity.Netcode.Samples;
 
 public class ServerScoreReplicator : NetworkBehaviour
 {
-    private NetworkVariable<int> m_ReplicatedScore = new NetworkVariable<int>();
+    NetworkVariable<int> m_ReplicatedScore = new NetworkVariable<int>();
+
+    public NetworkVariable<int> replicatedScore => m_ReplicatedScore; 
```

Comment:
Nitpick: being a property, it should be `ReplicatedScore` (link [here](https://github.cds.internal.unity3d.com/unity/unity-meta/blob/c0eeb02f04bdd06f0aedc0eee78e67ce20786c5d/ReferenceSource/CSharp/Assets/CSharpReference.cs#L194))